packages/animations/example/lib/main.dart

```diff
@@ -1,3 +1,84 @@
-import 'package:flutter/widgets.dart';
+import 'package:flutter/material.dart';
 
-void main() => runApp(const Placeholder());
+import 'shared_axis_transition/shared_axis_transition.dart';
```

Comment:
Do you think we'll have more demos to place inside the `shared_axis_transition` directory? Otherwise the directory probably doesn't have any benefit?

Reply:
That's true, I had originally anticipated needing to write multiple files in order to complete the demo, but it was simple enough to still be manageable in a single file.

I will just turn this into a file instead of it's own directory.